Providers/Profile.swift

```diff
@@ -1430,8 +1430,14 @@ open class BrowserProfile: Profile {
         }
 
         public func syncClientsThenTabs() -> SyncResult {
-            return self.sync("clients", function: self.syncClientsWithDelegate)
-            >>> { self.sync("tabs", function: self.syncTabsWithDelegate) }
+            return self.syncSeveral(
+                why: .user,
+                synchronizers:
+                ("clients", self.syncClientsWithDelegate),
+                ("tabs", self.syncTabsWithDelegate)) >>== { statuses in
+                let status = statuses.find { "tabs" == $0.0 }
+                return deferMaybe(status!.1)
+            }
```

Comment:
nit: Indentation

I will look odd on github but its more readable on the XCode side

```suggestion
            return self.syncSeveral(why: .user, synchronizers: ("clients", self.syncClientsWithDelegate),
                                                                                              ("tabs", self.syncTabsWithDelegate)) >>== {
                statuses in
                let status = statuses.find { "tabs" == $0.0 }
                return deferMaybe(status!.1)
            }
```

Reply:
I think it might be something like this because your first line will be too long.

```
            return self.syncSeveral(
                why: .user,
                synchronizers:
                ("clients", self.syncClientsWithDelegate),
                ("tabs", self.syncTabsWithDelegate)
            ) >>== { statuses in
                let status = statuses.find { "tabs" == $0.0 }
                return deferMaybe(status!.1)
            }
```

Reply:
yeah, that makes sense I just quickly typed on github and likely made it worse ha